.github/workflows/release-manual.yml

```
@@ -0,0 +1,69 @@
+      - name: Run tests
+        run: pnpm test
+      
+      - name: Run type checks
+        run: pnpm typecheck
+      
+      - name: Publish to npm
+        run: |
+          pnpm changeset publish --tag ${{ inputs.tag }}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)